### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by silviatofana**
Chose an API, [Covid API](https://disease.sh/v3/covid-19/countries).
Built files structure for the App.
Made routes and navigation for the app.
Made sure that list of items were displayed and filtered with data from the API.
Create tests.
Style components.
Deploy project.
Record video for project.
Create a good REDME file

### CODE REVIEW

File: github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (prolajumokeoni) [inline github/workflows/linters.yml:37]: - [ ] Kindly use the right [linters.yml](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml) config file content  for react-redux from the microverse linters config repository

File: src/__tests__/App.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+describe('App component', () => {
+  test('renders', () => {
+    const tree = render(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (prolajumokeoni) [inline src/__tests__/App.test.js:16]: - [ ] Kindly ensure tests are passing, currently, tests are failing 👍🏿  
Screenshot_348

File: src/pages/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import FetchStats from '../store/api';
+import { GetStats } from '../store/reducer';
+
+const HomePage = () => {
```
Comment (prolajumokeoni) [inline src/pages/HomePage.js:7]: - [ ] Good job filtering the incoming data, from api 👍🏿 Your home page should have a list of categories that could be filtered using given parameters, e.g. category name. Hint: Users should be able to filter by regions or you can create a search bar to filter countries alphabetically 😉

File: src/pages/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  const Africa = CountryStore.filter((item) => item.continent === 'Africa');
+
+  return (
+    <div className="homePage">
+      <h1 className="continent">Africa</h1>
+      <ul className="dataUL">
+        {
+        Africa.map((country) => (
+          <Link key={country.country} to={{ pathname: `/country/${country.country}` }}>
```
Comment (prolajumokeoni) [inline src/pages/HomePage.js:26]: - [ ] In your project, it's not possible to navigate to the detail page by clicking the individual category item, in the home page, please fix this, see video below 👍🏿 

https://user-images.githubusercontent.com/69638013/180864367-9d73ef52-6198-41e4-bef3-cf1e48e0da59.mp4

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @silviatofana, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @silviatofana        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
WOOOOWWWWWWW THANKSSSSSS

**Pull request merged**